src/backend/tasks_cpu/handlers/insights.py

```diff
@@ -82,6 +87,50 @@ def do_year_insights(kind: str, year: Year) -> Response:
     return make_response("")
 
 
+@blueprint.route("/backend-tasks-b2/math/do/insights/leaderboards/<kind>/<int:year>")
```

Comment:
Does this need to get setup on a cron job as well?

Reply:
Future PR